packages/fxa-auth-server/README.md

```diff
@@ -32,36 +32,43 @@ Linting will also be run for staged files automatically via Husky when you attem
 
 ## Testing
 
-Run tests with:
+Before running tests make sure the correct services are running. If auth server is running in pm2 weird behavior can ensue, so
+prior to running tests we recommend creating a clean slate by running `yarn stop`.
 
-    npm test
+Now we can start testing. To run unit tests:
 
-Run specific tests with the following commands:
+- `nx test-unit fxa-auth-server`
+  _Note this matches how auth server unit tests jobs in CI._
 
-```bash
-# Test only test/local/account_routes.js
-# Note: This command does not work for remote tests.
-npm test -- test/local/account_routes.js
+To run integration tests, we have to make sure databases and auxillary services required for integration testing are spun up.
+So run:
 
-# Grep for "SQSReceiver"
-NODE_ENV=dev npx mocha -r esbuild-register test/*/** -g "SQSReceiver"
-```
+- `yarn start infrastructure`
+- `nx start fxa-customs-server`
```

Comment:
Maybe this would be a good spot to explain why customs is needed?

Reply:
I thought about this, but people can look at the code. And the reason may change overtime. Customs is an auxiliary service.